src/main/resources/config/application-dev.yml

```diff
@@ -149,7 +150,8 @@ application:
     user-registration-channel-id:
   redis:
     # Add the following if you want to opt in to using Redis
-    # enabled: true
+    # Not using Redis
+    enabled: false
```

Comment:
I don't mind make it more specific. @calvinlu3 any thoughts why we didn't in the first place?